R/dataset.R

```diff
@@ -88,6 +88,42 @@ setMethod("notes<-", "CrunchDataset", function (x, value) {
     invisible(setEntitySlot(x, "notes", value))
 })
 
+
+#' Name, alias, and description for Crunch objects
+#'
+#' @param x a Dataset
+#' @param value For the setters, a single Variable to use as the primary key.
```

Comment:
Note that the variable must have all unique values. Also note that `NULL` is valid for assignment too. 

And "setter" should be singular.